api4/command.go

```diff
@@ -185,6 +185,31 @@ func listCommands(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(model.CommandListToJson(commands)))
 }
 
+func getCommand(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequireCommandId()
+	if c.Err != nil {
+		return
+	}
+
+	cmd, err := c.App.GetCommand(c.Params.CommandId)
+	if err != nil {
+		c.Err = err
+		return
+	}
+
+	// check for permissions to view this command; must have perms to view team and
+	// PERMISSION_MANAGE_SLASH_COMMANDS for the team the command belongs to.
+	if !c.App.SessionHasPermissionToTeam(c.App.Session, cmd.TeamId, model.PERMISSION_VIEW_TEAM) {
+		c.SetPermissionError(model.PERMISSION_VIEW_TEAM)
+		return
+	}
+	if !cmd.AutoComplete && !c.App.SessionHasPermissionToTeam(c.App.Session, cmd.TeamId, model.PERMISSION_MANAGE_SLASH_COMMANDS) {
```

Comment:
What's the rationale behind checking `cmd.AutoComplete` here?

Reply:
Great eye noticing that.  In the listCommand API it seems to use that flag to determine if the slash command is a custom command versus built-in.  It didn't look right to me either (why would autocomplete have anything to do with built-in vs custom).  I started a thread about it [here](https://community.mattermost.com/core/pl/8za4z1oyktyezk4x7qkwk6whhe).  I am trying to ensure that built-in commands are not modified.

Reply:
Ah I see, thanks for the explanation :). Does that mean you're basically checking if a command is builtin vs custom here? Maybe a simple comment just stating that will help the next person that comes around curious as to why is this odd check here :P

Reply:
Reviewing the ListXXXCommands implementations in app, I think I am/was misunderstanding why that flag was used to filter.  It is not determining if the command is custom.  I will remove from the code.

Reply:
Done